src/components/ResultsGrid.vue

```diff
@@ -24,6 +24,8 @@ export default {
     grid-template-columns: repeat(auto-fill, minmax(180px, 1fr)); /* Adapt number of columns to screen size */
     gap: 20px; /* Space between cards */
     padding: 20px;
+    max-width: 1200px;
+    margin: auto;
```

Comment:
`max-width` to set the grid to 2 rows of 5 at the largest. `auto` margin as a easy way to center an item whose parent is float instead of flex.